Modules/Filtering/Smoothing/include/itkDiscreteGaussianImageFilter.h

```diff
@@ -356,9 +356,8 @@ class ITK_TEMPLATE_EXPORT DiscreteGaussianImageFilter : public ImageToImageFilte
   /** Boundary condition use for the intermediate filters */
   RealBoundaryConditionPointerType m_RealBoundaryCondition;
 
- /** Default boundary condition use for the intermediate filters */
+  /** Default boundary condition use for the intermediate filters */
```

Comment:
This indentation (2 spaces) obviously looks better than the original (1 space) 👍  Do you know why the original wasn't fixed by clang-format initially?